src/signup.css

```diff
@@ -1,5 +1,7 @@
 body {
   font-family: "Montserrat", sans-serif;
+  display: flex;
```

Comment:
We usually don't want to target the body as a whole. You should add a centered container class to the main content element instead.

Reply:
ok, will work on it

Reply:
If you choose to do the way you're doing with flex, you will need a wrapper like this:

```
<main class="centered-container"> 
  <div> <--- you need this inner wrapper
    main content goes here
  </div>
</main>
```

Reply:
I've tried doing as such but it breaks the nav bar

Reply:
fixed !